docs/source/tutorials/playing-command.rst

```diff
@@ -0,0 +1,129 @@
+=========================
+Playing with your command
+=========================
+
+Now that you have started :doc:`first-plugin`, maybe you want to write a more
```

Comment:
We should consider that links to `:doc:` refs might look better with custom titles, at least to avoid the uppercase first letter that flows weirdly in the middle of a sentence.